package.json

```diff
@@ -161,7 +147,13 @@
     "watch": "1.0.2"
   },
   "dependencies": {
+    "autoprefixer": "^9.8.0",
+    "clean-css": "^4.2.3",
+    "cuid": "^2.1.4",
+    "find": "^0.3.0",
     "glob-stream": "^6.1.0",
+    "moment": "~2.22.2",
```

Comment:
We can remove `moment` from package.json, it's not used in scripts.

Reply:
Maybe we can do that in another PR to clean the dependencies

Reply:
I'm agree to clean all unused packages in another PR